src/pyinit.jl

```diff
@@ -47,6 +47,9 @@ function __init__()
             end
         end
         ccall((@pysym :Py_InitializeEx), Void, (Cint,), 0)
+
+        # Will get reinitialized properly on first use
+        is_windows() && (PyActCtx[] = C_NULL)
```

Comment:
Shouldn't this line be after the `end`, i.e. outside the `if !already_inited` block?